nautiluszim/scraper.py

```diff
@@ -345,26 +346,56 @@ def test_collection(self):
         for entry in self.json_collection:
             if not entry.get("files"):
                 continue
-            for relative_path in entry["files"]:
-                if relative_path not in all_names:
-                    missing_files.append(relative_path)
+            for file in entry["files"]:
+                path = self.parse_files(file)["path"]
+                if not path.startswith("http") and path not in all_names:
+                    missing_files.append(path)
 
         if missing_files:
             raise ValueError(
                 "File(s) referenced in collection but missing:\n - "
                 + "\n - ".join(missing_files)
             )
 
+    def parse_files(self, file: Union[str, Dict[str, str]]):
+        if isinstance(file, str):
+            return {"path": file, "filename": file}
```

Comment:
Given how it's used in the file, I think returning a `(path, filename)` tuple would simplify a bit